### ISSUE TITLE
request raspbian

### ISSUE THREAD

**Issue by Matschi89** [Robot/node-robot#25]
Hi there,

is there any chance robot-js will be available on the raspberry pi with raspbian?

thanks in advance
sincerely Matthias

**Comment by dkrutsko** [comment]
@Matschi89 I happen to have a few raspberry pi's laying around so I'd be happy to take a look. But I'm not quite sure what kind of changes would be required. Most of the functionality is based on standard POSIX and X11 routines. My understanding is that most of it should work fine, minus the compiling. For compiling, I'm hoping that [this PR](https://github.com/Robot/robot-js/pull/20) will fix those problems. Maybe if you have time you can try out that branch and see if it works? Otherwise, maybe see about [compiling](http://getrobot.net/docs/node.html#compiling) robot-js manually?

**Comment by Matschi89** [comment]
@dkrutsko thanks for the fast reply!
Unfortunately I'm a total newbie to everything programming related...
I'm afraid I can't be helpful.

I'm searching for a method to send key strokes/combination with node-red. 
There are some solutions, but they all rely on robot-js. 
Sorry for not being any helpful.. :/

**Comment by dkrutsko** [comment]
Perhaps [robotjs](https://github.com/octalmage/robotjs) (without the dash) might work better for you? Pardon the confusion, robotjs is actually a bit more portable when it comes to sending keys, etc. robot-js (this library) is focused on more advanced system automation.

**Comment by Matschi89** [comment]
Thanks for  pointing that out.
Unfortunately I get the same error when I try to install it

As far as I see (I'm pretty blind depending on programming topic) it fails trying to 
include <X11/Xutil.h>
I'll attach the logfile. Maybe that'll clear it.

[debug_log.zip](https://github.com/Robot/robot-js/files/1160320/debug_log.zip)

**Comment by dkrutsko** [comment]
Try running:

```bash
sudo apt-get install libxt-dev libxtst-dev libxinerama-dev -y
```

See [this](http://getrobot.net/docs/usage.html#linux) for more info

**Comment by Matschi89** [comment]
It worked so far! now trying to get a functional node-red node
Thanks!!

**Comment by karliky** [comment]
Seeing that robotjs is enough for this, Can we close this issue? 🔐 😄

**Comment by dkrutsko** [comment]
It works right @karliky, just need a binding.gyp?

**Comment by karliky** [comment]
Yeah, I think we'll solve a lot of problems with that gyp configuration file.

**Comment by dkrutsko** [comment]
Closing this issue, assuming it gets fixed with #35.

**Comment by dkrutsko** [comment]
Experimental support for this is coming with version 2.0.0.
